sql-jdbc/src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/ElasticsearchType.java

```diff
@@ -72,7 +72,7 @@
     IP(JDBCType.VARCHAR, String.class, 15, 0, false),
     NESTED(JDBCType.STRUCT, null, 0, 0, false),
     OBJECT(JDBCType.STRUCT, null, 0, 0, false),
-    DATE(JDBCType.TIMESTAMP, Timestamp.class, 24, 24, false),
```

Comment:
Are the changes in UT meant for this? I can see we're changing `date` type in test data and code to `timestamp` in UT?

Reply:
Yes, all timestamp type were taken as date type in JDBC, here it's corrected back to timestamp and so are the UTs